### PR TITLE
Migrated scenes to use the new SceneContent System

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Migrated as many scenes as possible to use the new SceneContent system where appropriate. Only scenes that reported minimal reserialization changes are included in this migration.

### CODE REVIEW

File: Assets/MRTK/Examples/Demos/Audio/Scenes/AudioLoFiEffectExamples.unity [hunk 1]
```diff
@@ -498,8 +498,9 @@ GameObject:
   serializedVersion: 6
   m_Component:
   - component: {fileID: 336154624}
+  - component: {fileID: 336154625}
   m_Layer: 0
-  m_Name: SceneContent
+  m_Name: MixedRealtiySceneContent
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/Audio/Scenes/AudioLoFiEffectExamples.unity:503]: slight typo in the GO names: MixedReal*ti*ySceneContent

Comment (RogPodge) [reply Assets/MRTK/Examples/Demos/Audio/Scenes/AudioLoFiEffectExamples.unity:503]: T_T

Comment (RogPodge) [reply Assets/MRTK/Examples/Demos/Audio/Scenes/AudioLoFiEffectExamples.unity]: fixed

File: Assets/MRTK/Examples/Demos/EyeTracking/Scenes/EyeTrackingDemo-01-BasicSetup.unity [hunk 2]
```diff
@@ -1308,7 +1322,7 @@ PrefabInstance:
         type: 3}
       propertyPath: m_isInputParsingRequired
       value: 1
-      objectReference: {fileID: 0}
+            objectReference: {fileID: 0}
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/EyeTracking/Scenes/EyeTrackingDemo-01-BasicSetup.unity:1325]: this indentation seems weird, but i'm not sure i know enough to be certain one way or another!

File: Assets/MRTK/Examples/Demos/EyeTracking/Scenes/EyeTrackingDemo-04-TargetPositioning.unity [hunk 3]
```diff
@@ -310,13 +324,13 @@ MonoBehaviour:
...
     spriteCount: 0
-    spaceCount: 27
-    wordCount: 25
+    spaceCount: 0
+    wordCount: 0
     linkCount: 0
-    lineCount: 5
-    pageCount: 1
+    lineCount: 0
+    pageCount: 0
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/EyeTracking/Scenes/EyeTrackingDemo-04-TargetPositioning.unity:333]: Intentionally included? Want to make sure there isn't any text being wiped out here

**Review by keveleigh — APPROVED** [review]

**Review by keveleigh — APPROVED** [review]

**Pull request merged**